src/utils.rs

```diff
@@ -29,3 +29,29 @@ mod tests {
         assert_eq!(b, 206);
     }
 }
+
+pub fn pos_to_voyager(x: u16, y: u16) -> usize {
+    // 0,  5   is left 1st row
+    // 6,  11  is left 2st row
+    // 12, 17  is left 3st row
+    // 18, 23  is left 4st row
+
+    // 24, 25 is left thumb keys
+    // 50, 51 is right thumb keys
+
+    // 26, 31  is right 1st row
+    // 32, 37  is right 2st row
+    // 38, 43  is right 3st row
+    // 44, 49  is right 4st row
+
+    #[rustfmt::skip]
+    let voyager_layout: [[usize; 12]; 5] = [
+        [ 0,  1,  2,  3,  4,  5,        26, 27, 28, 29, 30, 31],
+        [ 6,  7,  8,  9, 10, 11,        32, 33, 34, 35, 36, 37],
+        [12, 13, 14, 15, 16, 17,        38, 39, 40, 41, 42, 43],
+        [18, 19, 20, 21, 22, 23,        44, 45, 46, 47, 48, 49],
+
+        [60, 60, 60, 60, 24, 26,        50, 51, 60, 60, 60, 60]
```

Comment:
@BlackPhlox Pardon my late comment, but this line looks like it should read (25 instead of 26)
```diff
-[60, 60, 60, 60, 24, 26,        50, 51, 60, 60, 60, 60]
+[60, 60, 60, 60, 24, 25,        50, 51, 60, 60, 60, 60]
```

Reply:
Thank you! I've added it to PR #8